CHANGELOG.md

```diff
@@ -78,6 +76,10 @@ Bugsnag Notifiers on other platforms.
 * Add metadata accessor methods to `BugsnagEvent`
   [#465](https://github.com/bugsnag/bugsnag-cocoa/pull/465)
 
+* Added a method allowing merging arbitrary (JSON-encodable) dictionaries with 
+  an existing Event Metadata instance (`BugsnagMetadata.addMetadataToSection:values:`)
```

Comment:
What does this mean for users exactly? Saying that it "merges" dicts with Event Metadata could be unclear about what will happen when this method is called. Can you provide a code example here? The headerdoc for the function is fairly clear.

Reply:
I've expanded the message to have what I hope is enough clarity.  If you still think it would benefit from a code sample let me know.